### PR TITLE
clean up dos file carriage returns

### PR DESCRIPTION
**Pull request by gabemontero**
Fixes https://github.com/openshift/jenkins/issues/390

@openshift/devex fyi

thx for the `bash` assist @bparees 

validated with both dos and unix files

`dos2unix` was not available in the openshift/origin image

### CODE REVIEW

File: 2/contrib/jenkins/install-plugins.sh [hunk 1]
```diff
@@ -338,6 +338,11 @@ main() {
 
     mkdir -p "$REF_DIR" || exit 1
 
+    for file in $@; do
+	echo "Cleaning up file ${file}"
```
Comment (bparees) [inline 2/contrib/jenkins/install-plugins.sh]: can remove this.

**Comment by gabemontero** [comment]
it does work with empty lines in the file.... not sure what you mean by trailing empty lines

File: 2/contrib/jenkins/install-plugins.sh (hunk 1 above)
Comment (gabemontero) [reply 2/contrib/jenkins/install-plugins.sh]: echo converted to comment

**Comment by bparees** [comment]
>  not sure what you mean by trailing empty lines

just empty lines at the end of the file vs in the middle.  no reason i'd expect that to be different.

**Comment by bparees** [comment]
lgtm

**Comment by gabemontero** [comment]
[merge]

**Pull request merged**